hledger-reports.el

```diff
@@ -730,8 +730,7 @@ Optional parameter WIDTH decides the maximum width of a line."
 
 (defun hledger-compound-money (init periods periodic-rate)
   "Compound INIT amount for PERIODS units at PERIODIC-RATE.
-
-PERIODIC-RATE is in % and hence must be divided by 100."
+PERIODIC-RATE is a percentage."
```

Comment:
Is there a less ambiguous way to phrase this? For me, it’s always confusing when I see mention of percentages, because it’s unclear to me whether the function expects 99.9% to be passed as `99.9` or as `0.999`. (It looks like the earlier docstring incorrectly implied the latter, so this is an important correction.) Perhaps adding an example value would be enough to clarify it.

Reply:
Back when I wrote these functions I wasn't familiar with `calc-*` functions that I could have used for these calculations.

Reply:
I think I meant to say that the input number received is a percentage (e.g. 10 if we mean 10%) and to use it in the compound interest related calculations, it needs to be converted into a fraction (` 10/100`).